manifests/kuadrant/kuadrant-operator/operator.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: kuadrant-operator-catalog
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  sourceType: grpc
+  image: quay.io/kuadrant/kuadrant-operator-catalog:v0.11.0
```

Comment:
not something needed to be solved right now, but but I wonder how we might keep this in sync with each release of kuadrant itself. Ideally it would always install the latest released version

Reply:
I assume that once a stable release is hit, there would be a `stable` channel that just keeps adding new releases when they are out. If the subscription is set to `Automatic` new releases would be installed as soon as they are available in the catalog.
A different concern is to keep gateway-api CRDs and istio in sync. That would require custom logic. If kuadrant-operator provided somehow the version constraints (maybe a configmap?) for Istio and gateway-api we could create a Config Management Plugin for argocd to dynamically determine the versions to install. Same for envoy-gateway.
Just throwing out the idea, haven't really given it much thought ...